Sources/RswiftCore/Generators/ColorStructGenerator.swift

```diff
@@ -57,8 +57,13 @@ struct ColorStructGenerator: ExternalOnlyStructGenerator {
       implements: [],
       typealiasses: [],
       properties: colorLets,
-      functions: groupedColors.uniques.map { [ generateColorFunction(for: $0, at: externalAccessLevel, prefix: qualifiedName),
-                                               generateWatchOSColorFunction(for: $0, at: externalAccessLevel, prefix: qualifiedName)] }.flatMap { $0 },
+      functions: groupedColors.uniques.map { name -> [Function] in
```

Comment:
Curious if we can `flatMap` directly here like this:
```suggestion
      functions: groupedColors.uniques.flatMap { name -> [Function] in
```